spmd/tensor/parallel/api.py

```diff
@@ -181,8 +276,8 @@ def _colwise_parallelize_fn(name, module, device_mesh):  # pyre-ignore[2, 3]
             distribute_module(
                 m,
                 device_mesh,
-                _colwise_parallelize_fn,
-                input_fn=parallel_style._prepare_input  # type: ignore[arg-type, misc] # pyre-ignore[6]
```

Comment:
You should keep the order as it is, otherwise, mypy might fail.

Reply:
Must be overriding your CI fix by mistake when resolving merge conflict. Thx for pointing out!